### ISSUE TITLE
Maven build: Resource archive cannot be found (dcm4che-dict:jar:5.16.1)

### ISSUE THREAD

**Issue by CharlesGueunet** [dcm4che/dcm4che#422]
**Describe the bug**

Build failed during the `mvn generate-sources` required for Java 9 / 10 ( see #177 ) with the error:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-remote-resources-plugin:1.5:process (dictionary) on project dcm4che-core: Resources archive cannot be found.: Failure to find org.dcm4che:dcm4che-dict:jar:5.16.1 in http://maven.dcm4che.org was cached in the local repository, resolution will not be reattempted until the update interval of www.dcm4che.org has elapsed or updates are forced
```

And indeed, on the [maven.dcm4che website](http://maven.dcm4che.org/org/dcm4che/dcm4che-dict/) the latest archive is the 5.16.0

**To Reproduce**

1. Clone the repository
2. Go to the source folder
3. Execute `mvn generate-sources`

**Expected behavior**

The sources to be generated so that the `mvn install` will work.

**Desktop:**
 - OS: Linux 5.0.2
 - Version Oracle JKD-12, Java 9 & 10

**Additional context**

I have also tried with #394

**Comment by CharlesGueunet** [comment]
This problem does not occur when Java 8 is used

**Comment by gunterze** [comment]
Solved by fcf7e8b
